doc/usecases/elektra_web/UC_multiple_config.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Remote configuration of a multiple instances
+
+## Summary
+
+Title: Remote configuration of a multiple instances
+Scope: Configuration
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User configures multiple (standalone) instances via the client.
+
+## Scenarios
+
+Precondition: Add instance (multiple times)
+Main success scenario: User successfully changes configuration of multiple instances.
+Alternative scenario: Instance not online. The user is informed about the issue.
```

Comment:
Why is it possible to select offline nodes?

Better say: "A node got offline ..."